content/06.know-your-problem.md

```diff
@@ -7,10 +7,13 @@ If so, obtaining this data (and associated metadata), and reviewing the study pr
 This can help to ensure that data is as expected and can prevent the wasted time and effort that occur when issues are discovered later on in the analytic process.
 For example, a publication or resource might purportedly offer an appropriate dataset that is found to be inadequate upon acquisition.
 The data may be unstructured when it is supposed to be structured, crucial metadata such as sample stratification might be missing, or the usable sample size may be different than expected.
-Any of these data issues might limit a researcher's ability to use DL to address the biological question at hand, or might otherwise require adjustment before DL can be used.
+Any of these data issues might limit a researcher's ability to use DL to address the biological question at hand or might otherwise require adjustment before DL can be used.
 Data collection should also be carefully documented, or a data collection protocol should be created and specified in the project documentation.
+
 Information about the resources used, download dates, and dataset versions are critical to preserve.
-Doing so will help to minimize operational confusion and will increase the reproducibility of the analysis.
+Doing so will help to minimize operational confusion and will increase the reproducibility of the analysis. Best practices for reproducibility also include sharing your collected dataset and metadata along upon publication of your study, ideally in a public dataset repository if there are no ethical or privacy concerns and no copyright restrictions.

```

Comment:
```suggestion
Doing so will help to minimize operational confusion and will increase the reproducibility of the analysis.
Best practices for reproducibility also include sharing the collected dataset and metadata along upon publication of the study in a public dataset repository if there are no ethical or privacy concerns and no copyright restrictions.
```

Reply:
Thanks! I was/am in favor of avoiding 2nd person but wasn't sure if we adopted it. I believe I saw it in several places when editing, so we probably need to double check regarding this again.